modules/commons/src/test/java/org/apache/synapse/commons/json/JsonValueTest.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * <p>
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.commons.json;
+
+import junit.framework.TestCase;
+import org.apache.axiom.om.OMElement;
+import org.apache.axis2.context.MessageContext;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class JsonValueTest extends TestCase {
+
+    private static final String val_1 = "\"Hello world!\"";
+    private static final String val_2 = "null";
+    private static final String val_3 = "true";
+    private static final String val_4 = "false";
+    private static final String val_5 = "12.5";
+    private static final String val_6 = "-12.5";
+    private static final String val_7 = "123";
+    private static final String val_8 = "123abc";
+    private static final String val_9 = "task";
+    private static final String val_10 = "fun";
+    private static final String val_11 = "abc";
+    private static final String val_12 = "-abc";
+
+    public void testCase1() {
```

Comment:
This is not the way to name tests. The test name should indicate what you are testing for. You are trying out different values, and each value signifies some condition you are testing for. So the test name should reflect that.